data_collection/web/advanced.rst

```diff
@@ -4,6 +4,53 @@
 Advanced
 ========
 
+.. _data-collection-installing-tracking-code-via-snippet:
```

Comment:
Let's move installation article to a separate document. Advanced would otherwise contain only this section (reserved-names have been moved to JS docs in a separate PR).